docs/guide/installation-procedure.md

```diff
@@ -267,10 +267,12 @@ You can pass this warning and install Webots by clicking on the "More info" link
 
 #### From the Installation File
 
-1. Download the `webots-{{ webots.version.package }}.dmg` installation file from our [website](https://cyberbotics.com).
-2. Double click on this file.
-This will mount on the desktop a volume named "Webots" containing the "Webots" folder.
-3. Move this folder to your "/Applications" folder or wherever you would like to install Webots.
+It is better to download Webots using `curl` so that it doesn't get tagged as "downloaded from the Internet" and won't be blocked by macOS Gatekeeper:
+
+1. Open a Terminal app.
+2. Type `curl -L -O https://github.com/cyberbotics/webots/releases/download/{{ webots.version.package }}/webots-{{ webots.version.package }}.dmg`.
+3. From the finder, locate the downloaded `dmg` file, double-click on it. This will mount on the desktop a volume named `Webots` containing the `Webots` application.
```

Comment:
It seems Finder is usually written with capital letter and I tried to check how we usually write extension and it seems we use uppercase without quotes
```suggestion
3. From the Finder, locate the downloaded DMG file, double-click on it. This will mount on the desktop a volume named `Webots` containing the `Webots` application.
```